src/code-snippet/code-snippet.component.ts

```diff
@@ -58,7 +58,7 @@ export enum SnippetType {
 				(click)="toggleSnippetExpansion()"
 				type="button">
 				<span class="bx--snippet-btn--text">{{expanded ? translations.SHOW_LESS : translations.SHOW_MORE}}</span>
-				<ibm-icon-chevron-down16 class="bx--icon-chevron--down" [ariaLabel]="translations.SHOW_MORE_ICON"></ibm-icon-chevron-down16>
+				<ibm-icon-chevron-down16 class="bx--icon-chevron--down" [attr.aria-label]="translations.SHOW_MORE_ICON"></ibm-icon-chevron-down16>
```

Comment:
`[ariaLabel]` passes the label into the icon component and applies it to the svg, `[attr.aria-label]` simply applies the label to the host element ... `aria-label` is a valid attribute for `svg` so I'm not sure what's going on with the build error in #973

Reply:
@cal-smith you're perfectly correct. Sorry for missing these details and for the fuss! Obviously this PR had different intent when created, closing this. I'll look into #973 when have some time.
Cheers!

Reply:
Thanks!